CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed / Improved
 
 - Update sfui version to 0.7.11
+- Adjusted the error page (#398)
```

Comment:
please make pull from develop and add this to next hotfix section
```
## [1.0.3] - UNRELEASED

### Added
### Changed / Improved

- Adjusted the error page (#398)
```

Reply:
Done, Thanks!